lib/system/Contexts.hpp

```diff
@@ -34,7 +34,7 @@ namespace MAT_NS_BEGIN {
 
         IncomingEventContext(std::string const& id, std::string const& tenantToken, EventLatency latency, EventPersistence persistence, ::CsProtocol::Record* source)
             : source(source),
-            record{ id, tenantToken, latency, persistence },
+            record{ id, tenantToken, latency, persistence, source->cV },
```

Comment:
>source [](http://example.com/codeflow?start=59&length=6)

Potential crash - don't use source without checking it for null.

Reply:
Thanks. Missed it somehow. let me fix it